src/components/ArticleList.jsx

```diff
@@ -0,0 +1,88 @@
+/** @format */
+
+import { fetchAllArticles } from "../api/api";
+import { useState } from "react";
+import { useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+import { FaRegThumbsDown, FaRegThumbsUp } from "react-icons/fa";
+
+function ArticleList() {
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		fetchAllArticles().then((res) => {
+			const data = res.data.articles;
+			setArticles(data);
+		});
+	}, [setArticles]);
+
+	return (
+		<div
+			className="article-grid"
+			style={{
+				paddingTop: "80px",
+				paddingLeft: "50px",
+				paddingRight: "50px",
+				paddingBottom: "90px",
+			}}
+		>
+			<ul>
+				{articles.map(
+					({
+						article_id,
+						title,
+						topic,
+						author,
+						body,
+						created_at,
+						votes,
+						article_img_url,
+						comment_count,
+					}) => {
+						let isVotes = "Vote";
+						if (votes > 0) {
+							isVotes = { votes };
+						}
+
+						return (
+							<ArticleCard key={article_id}>
+								<section
+									className="each-card"
```

Comment:
Good work! However, this component is a little bloated - I'm somewhat unsure as to why this stuff (lines 49-78) isn't declared within the ArticleCard component?